packages/markdown/package.json

```diff
@@ -35,6 +44,7 @@
     "eslint": "^6.5.0",
     "jest": "^24.8.0",
     "jsinspect": "^0.12.7",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "webpack": "^3.10.0"
```

Comment:
Want to upgrade to Webpack 4 before merging this in? 😬

Reply:
I've tried that route. The update works fine internally, but when I try to `npm link` the explorer in to ReadMe and `start` the Angular app, the compilation fails:

```
ERROR in ../api-explorer/packages/markdown/dist/main.js
  Module build failed: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3"
```

And I'm hesitant to start messing with ReadMe's webpack configuration...

Reply:
(See also: [this discussion](https://github.com/readmeio/api-explorer/pull/398#discussion_r368046025) about updating `sass-loader`, above.)

Reply:
Per my other comment, we can hold off on this until either v5 is out, or ReadMe is on Webpack 4.